citrination_client/client.py

```diff
@@ -17,20 +17,23 @@ class CitrinationClient(object):
     Class for interacting with the Citrination api.
     """
 
-    def __init__(self, api_key, site='https://citrination.com'):
+    def __init__(self, api_key, site='https://citrination.com', proxies=None):
```

Comment:
Why not have the default be an empty map rather than `None`?  It'll save having to do the explicit check later on.

Reply:
I would want to keep the check in case a user passed in `proxies=None`

Reply:
Does that actually have consequences?  Does it `requests` to pass `None` rather than empty map?

Reply:
It turns out that both `{}` and `None` work just fine with requests. I just removed the if/else check and just pass through the user input, defaulting to None.